--- a/Makefile
+++ b/Makefile
@@ -2,9 +2,9 @@
 # Edited for Debian GNU/Linux.
 #BINDIR = /usr/X11R6/bin
 DESTDIR =
-BINDIR = /usr/local/bin
+BINDIR = /usr/bin
 DOCDIR = /usr/share/doc/xppaut
-MANDIR = /usr/local/man/man1
+MANDIR = /usr/share/man/man1
 # End Debian Edit
 #################################
 #
@@ -20,7 +20,7 @@ HELP=help/*.html
 # Use Gnu compiler
 #CC= gcc
 # Use c99 to compile according to newer ISO C standards (IEEE Std 1003.1-2001)
-CC= gcc -std=c99 -pedantic -D_XOPEN_SOURCE=600 -Wall  
+CC= gcc -O -std=c99 -pedantic -D_XOPEN_SOURCE=600 -Wall  
 #AUTLIBS=  -lsundials_cvode -lX11 -lm 
 #These are the 32bit compat libraries.
 AUTLIBS=  -lm 
@@ -36,9 +36,9 @@ AUTLIBS=  -lm
 #CFLAGS=   -g -O -m32 -DNON_UNIX_STDIO -DAUTO -DCVODE_YES  -DHAVEDLL -DMYSTR1=$(MAJORVER) -DMYSTR2=$(MINORVER)  -I/usr/X11R6/include
 #CFLAGS=   -g -O -m64 -DNOERRNO -DNON_UNIX_STDIO -DAUTO -DCVODE_YES  -DHAVEDLL -DMYSTR1=$(MAJORVER) -DMYSTR2=$(MINORVER)  -I/usr/include/X11
 
-CFLAGS= -g -pedantic -O -m32 -DNOERRNO -DNON_UNIX_STDIO -DAUTO -DCVODE_YES  -DHAVEDLL -DMYSTR1=$(MAJORVER) -DMYSTR2=$(MINORVER)  -I/usr/include/X11 
+CFLAGS = -g -pedantic -O -DNOERRNO -DNON_UNIX_STDIO -DAUTO -DCVODE_YES  -DHAVEDLL -DMYSTR1=$(MAJORVER) -DMYSTR2=$(MINORVER)  -I/usr/include/X11 
 #LDFLAGS=  -m64 -L/usr/lib -L/usr/lib64
-LDFLAGS=  -m32 -L/usr/X11R6/lib 
+LDFLAGS=  -L/usr/lib 
 LIBS=  -lX11 -lm -ldl   
 # NOTE: Recent (RedHat 8) versions of GCC seem to no longer have
 # the integer errno, so compile with the -DNOERRNO option as well
@@ -266,7 +266,7 @@ install: xppaut
 	install -m 755 xppaut $(DESTDIR)$(BINDIR)
 	cp -r ode* $(DESTDIR)$(DOCDIR)/examples
 	cp -r help/* $(DESTDIR)$(DOCDIR)/html
-	cp README *.pdf $(DESTDIR)$(DOCDIR)
+	#cp README *.pdf $(DESTDIR)$(DOCDIR)
 	cp *.xbm $(DESTDIR)$(DOCDIR)/xbm 
 	
 	cp xppaut.1 $(DESTDIR)$(MANDIR)
