Index: xppaut-5.98/Makefile
===================================================================
--- xppaut-5.98.orig/Makefile	2009-05-08 15:22:41.000000000 -0400
+++ xppaut-5.98/Makefile	2009-05-08 15:26:13.000000000 -0400
@@ -1,7 +1,7 @@
 # Copyright (C) 1990-2006 Bard Ermentrout
 # Edited for Debian GNU/Linux.
 DESTDIR =
-BINDIR = $(DESTDIR)/usr/X11R6/bin
+BINDIR = $(DESTDIR)/usr/bin
 DOCDIR = $(DESTDIR)/usr/share/doc/xppaut
 # End Debian Edit
 #################################
@@ -21,9 +21,9 @@
 # Standard Linux distributions   #
 ##################################
 #CFLAGS=   -g -O -DAUTO -DCVODE_YES -DHAVEDLL -DMYSTR=$(VERSION)  -I/usr/X11R6/include
-#CFLAGS=   -g -O -DAUTO -DCVODE_YES  -DHAVEDLL -DMYSTR=$(VERSION)  -I/usr/X11R6/include
-#LDFLAGS=  -L/usr/X11R6/lib
-#LIBS= -lX11 -lm -ldl 
+CFLAGS += -g -O -DAUTO -DCVODE_YES  -DHAVEDLL -DMYSTR=$(VERSION)  -I/usr/include
+LDFLAGS = -L/usr/lib
+LIBS = -lX11 -lm -ldl 
 # NOTE: Recent (RedHat 8) versions of GCC seem to no longer have
 # the integer errno, so compile with the -DNOERRNO option as well
 #
@@ -37,9 +37,9 @@
 #################################
 # MACOSX                        #
 #################################
-CFLAGS=   -g -O -DMACOSX -DAUTO -DCVODE_YES  -DMYSTR=$(VERSION) -I/usr/X11R6/include
-LIBS=  -lX11 -lm
-LDFLAGS=  -L/usr/X11R6/lib
+#CFLAGS=   -g -O -DMACOSX -DAUTO -DCVODE_YES  -DMYSTR=$(VERSION) -I/usr/X11R6/include
+#LIBS=  -lX11 -lm
+#LDFLAGS=  -L/usr/X11R6/lib
 #
 #################################
 # CYGWIN                        #
@@ -190,20 +190,19 @@
 	mkdir -p $(BINDIR)
 	mkdir -p $(DOCDIR)/html
 	mkdir -p $(DOCDIR)/examples
-	mkdir -p $(DESTDIR)/usr/X11R6/man/man1
+	mkdir -p $(DESTDIR)/usr/share/man/man1
 # Put everything home
-	strip xppaut
 	cp xppaut $(BINDIR)
 	cp -r ode* $(DOCDIR)/examples
 	cp -r help/* $(DOCDIR)/html
 	cp README *.pdf $(DOCDIR) 
-	cp xppaut.1 $(DESTDIR)/usr/X11R6/man/man1
+	cp xppaut.1 $(DESTDIR)/usr/share/man/man1
 # End Debian Ed
 uninstall: 
 # Remove everything you created
 	rm $(BINDIR)/xppaut
 	rm -r $(DOCDIR)
-	rm -r $(DESTDIR)/usr/X11R6/man/man1/xppaut.1
+	rm -r $(DESTDIR)/usr/share/man/man1/xppaut.1
 # End Debian Ed
 ##################################################
 #    Make s stand alone library -must link your rhs
