diff --git a/menudrive.c b/menudrive.c
index 0973452..1d43431 100644
--- a/menudrive.c
+++ b/menudrive.c
@@ -2,6 +2,7 @@
 
 #include <sys/wait.h>
 #include <unistd.h>
+#include <sys/stat.h>
 /* the menu for XPP commands 
    this calls any command
    it also has lots of the direct X Gui stuff
@@ -118,20 +119,28 @@ void xpp_hlp()
 {
 
   char cmd[256];
-
-  if(getenv("XPPHELP")==NULL)
-  {
-  	err_msg("Environment variable XPPHELP undefined.");
-    	return;
+  char *helpPath;
+  helpPath = getenv("XPPHELP");
+
+  if(helpPath==NULL){
+    struct stat* statBuff = (struct stat*) malloc(sizeof(struct stat));
+    /* This is the default path on the Debian GNU/Linux system: */
+    stat ("/usr/share/doc/xppaut/html", statBuff);
+    if (S_ISDIR (statBuff->st_mode))
+      helpPath = "/usr/share/doc/xppaut/html";
+    else {
+  	  err_msg("Environment variable XPPHELP undefined.");
+      return;
+    }
   }
-  
+
   if (getenv("XPPBROWSER")==NULL)
   {
     	err_msg("Environment variable XPPBROWSER undefined.");
     	return;
   }
   
-  sprintf(cmd,"file:///%s",getenv("XPPHELP"));
+  sprintf(cmd,"file:///%s",helpPath);
   
   if(fork()==0){
     
